code-challenges.test.js

```diff
@@ -37,7 +37,7 @@ const fibLength2 = 10
 // b) Create the function that makes the test pass.
 const fibSeq = (element) =>{
```

Comment:
Challenge 1:
Nice logic, but your pseudo code does not mention how you are getting those new numbers that are getting pushed into the array you made.

Challenge 2:
Your pseudo code is not descriptive of what your code is doing. Your function also has a couple things preventing it from working properly. The sort function should be `.sort((a,b) => a-b`. Also, Object.entries will not give you an array of values, but it will store each key value pair in an array, which gives you an array of arrays.

Challenge 3:
Here again your pseudo code does not explain your process. Under process, you have "create a variable and update it as we iterate over the array", which is not very descriptive. Update it how? You are also using `.map`, so "updating" would not be an accurate word to describe what is happening. Work on your pseudo code!